staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -790,47 +799,62 @@ func (qs *queueSet) finishRequestLocked(r *request) {
 		metrics.AddRequestConcurrencyInUse(qs.qCfg.Name, r.fsName, -r.Seats())
 		if r.queue != nil {
 			r.queue.seatsInUse -= r.Seats()
-
-			if klog.V(6).Enabled() {
-				klog.Infof("QS(%s) at r=%s v=%.9fs: request %#+v %#+v finished, adjusted queue %d virtual start time to %.9fs due to service time %.9fs, queue will have %d waiting & %d executing",
-					qs.qCfg.Name, now.Format(nsTimeFmt), qs.virtualTime, r.descr1, r.descr2, r.queue.index,
-					r.queue.virtualStart, S, r.queue.requests.Length(), r.queue.requestsExecuting)
-			}
 		}
 	}
 
 	defer func() {
 		if r.workEstimate.AdditionalLatency <= 0 {
 			// release the seats allocated to this request immediately
 			releaseSeatsLocked()
+			if !klog.V(6).Enabled() {
```

Comment:
It's a slightly weird pattern to me, but it's harmless, so I'm not going to block on it.

[In general, I don't think logs at 4+ verbosity are useful, because there are so many of them that it's impossible to find anything there anyway. But we can follow up on it separately.]